manage.py

```diff
@@ -89,7 +89,7 @@ def setup_dev():
                   .filter_by(permissions=Permission.AGENCY_WORKER)
                   .first(),
                   confirmed=True)
-    worker.agency = Agency.query.filter_by(name='SEPTA').first()
+    worker.agencies = [Agency.query.filter_by(name='SEPTA').first()]
```

Comment:
Could you explain what's going on here? My instinct would be that we should have a constant defined for the `SEPTA` string, but I'm not quite sure what purpose it's serving so I could be wrong.